### DISCUSSION TITLE
Spaces between pages

### DISCUSSION THREAD

**Discussion by athloncore1** [sumatrapdfreader/sumatrapdf#3245] [Ideas]
![book view](https://user-images.githubusercontent.com/123800497/215260125-b887ccad-438a-48d7-8823-f444d6ff1cea.png)

Hello
I'm reviewing files with Book View mod.
Can you connect the spaces between the pages?

**Comment by GitHubRulesOK** [comment]
Probably ! the space can be reduced by edit in advanced options `PageSpacing = 0 4` either for PDF (FixedPageUI) and/or for Manga/Images (ComicBookUI). DO NOT use negative values.
![image](https://user-images.githubusercontent.com/12754913/215267053-e6907602-562c-4af4-bfa3-ad549d137ad3.png)

**Comment by athloncore1** [comment]
Thank you.

I made the settings you said in the last version. Sumatra pdf is a very fast running program.
